src/ScottPlot5/ScottPlot/Control/MouseButton.cs

```diff
@@ -11,5 +11,9 @@ public enum MouseButton
         Left,
         Right,
         Middle,
+        WheelUp,
```

Comment:
I don't think I like considering mouse wheel movements to `MouseButton`s.

I know a `MouseWheelDirection` enum is a little verbose, but I think it's a better option.

Reply:
The gotcha here is that a wheel event is being treated like a button press

https://github.com/ScottPlot/ScottPlot/blob/d79f1898eda169daf884077ef9c6fb1e0c46a269/src/ScottPlot5/ScottPlot/Control/Backend.cs#L97-L101

I think you're right though that wheel events should be distinct, and probably not passed into the MouseDown function.

However this gets redesigned, it should be easy to re-write mouse wheel events to do different things (like pan)

Reply:
In #1982 I refactored this to be separate enumerations, `MouseButton` and `MouseWheelDirection`